include/cppkafka/utils/backoff_performer.h

```diff
@@ -57,11 +57,26 @@ class CPPKAFKA_API BackoffPerformer {
     };
 
     /**
-     * Constructs an instance of backoff perform
+     * Constructs an instance of backoff performer
      * 
      * By default, the linear backoff policy is used
      */
     BackoffPerformer();
+    
+    /**
+     * \brief Constructs an instance of backoff performer
+     *
+     * \param initial_backoff The initial backoff interval
+     * \param backoff_step The backoff step interval
+     * \param maximum_backoff The maximum backoff interval
+     * \param policy The backoff policy
+     * \param maximum_retries The max number of retries
+     */
+    BackoffPerformer(TimeUnit initial_backoff,
```

Comment:
Looks like you only removed this constructor on the committer, this one's still here.